rust/chains/tw_greenfield/fuzz/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "tw_greenfield-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+tw_any_coin = { path = "../../../tw_any_coin", features = ["test-utils"] }
+tw_coin_registry = { path = "../../../tw_coin_registry" }
+tw_proto = { path = "../../../tw_proto", features = ["fuzz"] }
+
+[dependencies.tw_greenfield]
+path = ".."
+
+# Prevent this from interfering with workspaces
+[workspace]
```

Comment:
What's happening here?

Reply:
`fuzz/Cargo.toml` is generated by calling `cargo fuzz init` command.
So this is a workaround for the `tw_greenfield/fuzz` to not be affected by the parent workspace.